.gitignore

```diff
@@ -11,4 +11,5 @@ report.md
 # Libraries don't need dependency lock
 # Dependencies will be locked in application that uses them
 /.deps.lock
+shard.lock
```

Comment:
this is a part of the repo, let's remove it from gitignore.

Reply:
ok; it ended on my pc when I did "shards install" in order to run spec